src/pages/SignUp/SignUp.loader.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react'
+import { Route } from 'react-router'
+
+import PageLoader, { ReducerDescription } from 'components/PageLoader'
+
+import { SignUpLayout } from './SignUp.layout'
+
+export class SignUp extends PageLoader<any, object> {
+  readonly component = SignUpLayout
+
+  initialState(): object {
+    return {}
+  }
+
+  routeHook(key: string): JSX.Element[] {
+    return [<Route path="/register" key={key} component={this.container()} />]
```

Comment:
Why not make the route `sign-up` for consistency? (components and other stuff seem to be named `sign-up`)

Also: I find `sign up` matches more nicely to `sign in` and `register` does to `log in`. But that's just my personal preference.

Reply:
I was going back and forth on updating everything to either be register or sign-up. I'll change this to sign-up for now. As far as the pairing I'm not sure that I have a real preference there, but in the Invision design they use `Sign Up` / `Log In`